packages/react-composites/src/composites/CallComposite/index.ts

```diff
@@ -14,5 +14,41 @@ export type {
   CustomCallControlButtonProps,
   CustomControlButtonProps
 } from './types/CallControlOptions';
-export * from './adapter';
```

Comment:
This must have been painful. Was there an automated way to do this? Or did you have to manually export each one individually?